server/src/main/java/org/gluu/oxtrust/action/UpdatePersonAction.java

```diff
@@ -687,15 +686,26 @@ private boolean validatePerson(GluuCustomPerson person) throws Exception {
 	
 		GluuCustomPerson  gluuCustomPerson  = personService.getPersonByUid(person.getUid());
 		if (gluuCustomPerson != null){
+			if(!update){
```

Comment:
Can you move all messages from UpdatePersonAction.java to message resources file